core/modules/filters/in-tagtree-of.js

```diff
@@ -0,0 +1,71 @@
+/*\
+title: $:/core/modules/filters/in-tagtree-of.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for checking if tiddlers are in a tag tree with a specified root
+
+Finds out where a tiddler originates from, is it in a tag tree with xxx as root?
+
+Based on:
+- https://github.com/bimlas/tw5-kin-filter/blob/master/plugins/kin-filter/kin.js
+- https://talk.tiddlywiki.org/t/recursive-filter-operators-to-show-all-tiddlers-beneath-a-tag-and-all-tags-above-a-tiddler/3814
+
+\*/
+
+"use strict";
+
+/*
+Export our filter function
+*/
+exports["in-tagtree-of"] = function(source,operator,options) {
+	const rootTiddler = operator.operand;
+	/*
+	 * By default we check tiddler passed-in is tagged with the operand (or is its child), we output the tiddler passed-in, otherwise output empty.
+	 * But if `isInclusive` is true, if tiddler operand itself is passed-in, we output it, even if the operand itself is not tagged with itself.
+	 */
```

Comment:
You should replace the "Export our filter function" comment at line 19 with the 2 comments here.